esp32-wroom-rp/src/gpio.rs

```diff
@@ -131,13 +133,13 @@ where
 
     fn wait_for_esp_ready(&self) {
         while !self.get_esp_ready() {
-            //cortex_m::asm::nop(); // Make sure rustc doesn't optimize this loop out
+            hint::spin_loop(); // Make sure rustc doesn't optimize this loop out
```

Comment:
This is very cool

Reply:
Yes indeed, also architecture independent!